WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModelProtocol.swift

```diff
@@ -224,4 +226,8 @@ extension ProductFormViewModelProtocol {
             return true
         }
     }
+
+    func updateProductCustomFields(customFields: [MetaData]) {
+        // Default implementation does nothing
+    }
```

Comment:
Please let me know if you have something against using a default implementation here instead of a `no-op` override in the `ProductVariationFormViewModel`